src/routes/+page.svelte

```diff
@@ -226,7 +226,7 @@
 						type="text"
 						bind:value={profileDirectory}
 						id="profile-directory"
-						placeholder="Leave blank to let the installer decide"
+						placeholder="(default .minecraft folder path)"
```

Comment:
maybe a little confusing since the default is `.minecraft/fabulously-optimized-$MCVERSION`